app/src/main/java/umc/cozymate/ui/splash/SplashActivity.kt

```
@@ -173,11 +174,11 @@ class SplashActivity : AppCompatActivity() {
                     try {
                         splashViewModel.setTokenInfo(result.body()!!.result.tokenResponseDTO)
                         splashViewModel.saveToken()
-
                         splashViewModel.memberCheck()
                         splashViewModel.isMember.observe(this) { isMember ->
                             if (isMember == true) goCozyHome()
                             else if (isMember == false) goOnboarding()
+                            else if (isMember == null) {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyIfBlock)